connectors/sources/mysql.py

```diff
@@ -106,11 +99,44 @@ async def close(self):
         await self.connection_pool.wait_closed()
         self.connection_pool = None
 
+    def _validate_connection_fields(self):
```

Comment:
we can use it to validate any config, we could call it `_validate_configuration`

Reply:
Done